Ark.Tools.Auth0/AuthenticationApiClientCachingDecorator.cs

```diff
@@ -62,10 +65,10 @@ private static TimeSpan _expiresIn(string accessToken)
             var decode = new JwtBuilder()
                                 .DoNotVerifySignature()
                                 .WithAlgorithm(new HMACSHA256Algorithm())
-                                .Decode<IDictionary<string, object>>(accessToken);
+                                .Decode<Token>(accessToken);
 #pragma warning restore CS0618 // Type or member is obsolete
 
-            var res = DateTimeOffset.FromUnixTimeSeconds((long)decode["exp"]) - DateTimeOffset.UtcNow;

```

Comment:
Why there a reason for the cast to long given that now there is a custom object with defined types?